### PR TITLE
Use new R 3.2 download methods (wininet and libcurl) when available

### PR DESCRIPTION
**Pull request by jjallaire**
@jcheng5

### CODE REVIEW

File: R/download.r [hunk 1]
```diff
@@ -37,27 +38,50 @@
...
-      on.exit(suppressWarnings(seti2(internet2_start)))
-
-      # Needed for https. Will get warning if setInternet2(FALSE) already run
-      # and internet routines are used. But the warnings don't seem to matter.
-      suppressWarnings(seti2(TRUE))
-      download.file(url, ...)
-
+      
+      if (isR32) {
+        method <- "wininet"
```
Comment (jcheng5) [inline R/download.r]: Does wininet need to be tested for using capabilities() as well?

Comment (jcheng5) [inline R/download.r]: The answer is no. :)

Comment (jjallaire) [inline R/download.r]: No, there is no capabilities for "wininet" it's always there on R for
Windows 3.2

**Comment by wch** [comment]
Thanks!

**Pull request merged**